### ISSUE TITLE
RememberMe Authentication is always set

### ISSUE THREAD

**Issue by georgeconstantinou** [CakeDC/users#490]
When trying to disable remember me functionality via the config:

```php
'Users.RememberMe.active' => false
```

CakePHP throws Fatal Error:

```
Error: Call to a member function read() on null 
File /vendor/cakedc/users/src/Auth/RememberMeAuthenticate.php 
Line: 36
```

The temporary workaround we are using is to override more configuration options like so:

```php
'Users.RememberMe.active' => false,
'Auth.authenticate' => [
    'all' => [
        'finder' => 'auth',
    ],
    'CakeDC/Users.ApiKey',
    // 'CakeDC/Users.RememberMe',
    'Form'
]
```

**Comment by georgeconstantinou** [comment]
Probably related to https://github.com/CakeDC/users/issues/488

**Comment by steinkel** [comment]
closed per #489
